### DISCUSSION TITLE
How to select autocomplete item by mouse?

### DISCUSSION THREAD

**Discussion by KrzysztofDusko** [AvaloniaUI/AvaloniaEdit#364] [Q&A]
at this time mouse click closes complete window and do not insert clicked text.. (WPF version behave different).
![image](https://github.com/AvaloniaUI/AvaloniaEdit/assets/69449360/384c87ea-aada-4d24-8130-949c010216df)

**Comment by jmkinzer** [comment]
FWIW I would love to see this work as well.
